fe/fe-core/src/main/java/com/starrocks/system/SystemInfoService.java

```diff
@@ -787,6 +776,11 @@ public ImmutableCollection<ComputeNode> getBackends(boolean needAlive) {
         return ImmutableList.copyOf(backends);
     }
 
+    // for debug
```

Comment:
if it's only for debug, better add your github id in the comment in case forgotten